src/verify.cc

```diff
@@ -135,5 +137,17 @@ Status verifyJwt(const Jwt& jwt, const Jwks& jwks, int64_t now) {
                          : Status::JwksKidAlgMismatch;
 }
 
+Status verifyJwt(const Jwt& jwt, const Jwks& jwks, const jwt_audience_list &audiences) {
+  return verifyJwt(jwt, jwks, audiences, absl::ToUnixSeconds(absl::Now()));
+}
+
+Status verifyJwt(const Jwt& jwt, const Jwks& jwks, const jwt_audience_list &audiences, int64_t now) {
+  auto iter = std::find_first_of(jwt.audiences_.begin(), jwt.audiences_.end(), audiences.begin(), audiences.end());
```

Comment:
Try to use this check_audience class.  https://github.com/google/jwt_verify_lib/blob/master/jwt_verify_lib/check_audience.h

Reply:
Fixed in latest commit.